pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -424,6 +436,14 @@ func (tr *TaskRun) IsCancelled() bool {
 	return tr.Spec.Status == TaskRunSpecStatusCancelled
 }
 
+// GetStatusMessage function returns the status message is one is set on the TaskRun
+// If nothing is set as status message, GetStatusMessage returns an empty string.
+func (tr *TaskRun) GetStatusMessage() TaskRunSpecStatusMessage {
```

Comment:
same comment here, I don't think this function is necessary

Reply:
Done. Removed the function.